code/game/gamemodes/malfunction/malfunction.dm

```diff
@@ -267,8 +267,8 @@
 		world << "<B>The AI has chosen not to detonate the station!</B>"
 
 	else if (!station_captured &&                station_was_nuked)
-		feedback_set_details("round_end_result","halfwin - everyone killed by nuke")
-		world << "<FONT size = 3><B>Neutral Victory</B></FONT>"
+		feedback_set_details("round_end_result","draw - everyone killed by nuke")
+		world << "<FONT size = 3><B>Draw</B></FONT>"

```

Comment:
The idea is that the AI CAN nuke, it doesn't have to. The victory is more in the choice of it rather than the act of it. At least that's how I see it.